forta-agents-linked/src/agent.js

```diff
@@ -18,11 +18,17 @@ function provideHandleTransaction(timeHandler) {
   return async function handleTransaction(txEvent) {
     const findings = [];
     const filtered = txEvent.filterLog(eventABI, contractAddress);
+    let aboveThreshold = [];
     filtered.forEach((tx) => {
-      timeHandler.addToList(tx.args.agentId);
+      const { name } = tx;
+      if (name == "AgentUpdated") {
+        timeHandler.addToListUpdated(tx.args.agentId);
+      } else if (name == "Link") {
+        timeHandler.addToListLinked(tx.args.agentId);
+        aboveThreshold = timeHandler.checkIfPassedThreshold();
```

Comment:
we're only checking if the threshold is passed for Link events, but what if the Link event never fires? we would want to be alerted about this case i.e. agent is created, but does not get linked for longer than threshold